src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final int maxValue = 100;
+    private final Random number = new Random();
+    private final ColorSupplier colour = new ColorSupplier();
```

Comment:
Let's pass these dependencies to the constructor, as i suggested in the message in the group chat